src/CommunityToolkit.Maui/Converters/IsStringNullOrEmptyConverter.cs

```diff
@@ -30,7 +35,4 @@ public class IsNullOrEmptyConverter : ValueConverterExtension, ICommunityToolkit
 	/// <returns>N/A</returns>
 	public object? ConvertBack(object? value, Type? targetType, object? parameter, CultureInfo? culture)
 		=> throw new NotImplementedException();
```

Comment:
This probably isn't within the scope of this PR but I wanted to ask your thoughts on `NotImplementedException` vs `NotSupportedException`?

Clearly I have a view 😃  `NotSupportedException` makes it clear it isn't supported and probably won't be `NotImplementedException` can give the impression it might be supported/implemented at some point.

Reply:
Good point - I agree! 

Could you open a Proposal to change all of the Converters from using 
`NotImplementedException` to using `NotSupportedException`?

Maybe we also update the Contribution Guide and (if it's possible) add an `.editorconfig` rule to remove `NotImplementedException` since it implies that we forgot to import a feature?

Reply:
It is open at: https://github.com/CommunityToolkit/Maui/issues/292